geniesp/bpc_redcap_export_mapping.py

```diff
@@ -532,9 +533,11 @@ class BpcProjectRunner(metaclass=ABCMeta):
         "production":"syn21446571",
         "staging": "syn64018293"
     }
-    # main GENIE release folder (15.4-consortium)
+    # main GENIE release folder
     # Must use consortium release, because SEQ_YEAR is used
-    _MG_RELEASE_SYNID = "syn53170398"
+    # NOTE: Must match release tracking sheet and release table info
```

Comment:
Should we link out to the release tracking sheet and table info here?

Reply:
I think it's already in the [SOP](https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2621931648/GENIE+BPC+Pipeline+SOP#4.-Processing%3A-update-references) so should be good on this front